content/chapters/software-stack/lecture/slides/types-of-software.md

```diff
@@ -0,0 +1,62 @@
+---
+
+## Types of Software
+
+----
+
+### Usable vs Reusable
+
+* Software provides features by extending hardware
+* Features are used by:
+  * users (as interactive applications / processes)
+  * system (as non-interactive services / processes)
+  * other applications (as reusable libraries)
+* Software components
+  * Applications: usable
+  * Libraries: reusable
+
+----
+
+### Applications
+
+![Applications](media/applications.svg)
+
+----
+
+### Libraries
+
+![Libraries](media/libraries.svg)
+
+----
+
+### Application Software and System Software
+
+![Application and System Software](media/application-system-software.png)
```

Comment:
This image looks unattractive. It is correct but it just looks terrible. See if you can find a more appealing alternative.